README.rst

```diff
@@ -86,6 +86,24 @@ step name.
 
 .. note:: Artifacts from previous steps are not available within remote builds
 
+There are two image builders in ``buildrunner``. ``docker-py`` is the default
```

Comment:
I think this should not refer to docker-py or python-on-whales, but instead of state that the legacy builder is equivalent to `docker build` and the new one uses `docker buildx`.

Reply:
Also we should state the legacy builder will be removed in a future release.

Reply:
I agree. I re-worded it in the README